### PR TITLE
feat(exporters): working loki exporter 

### PR DESCRIPTION
**Pull request by kristofgyuracz**

### CODE REVIEW

File: api/telemetry/v1alpha1/collector_types.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 type CollectorSpec struct {
 	TenantSelector   metav1.LabelSelector `json:"tenantSelector,omitempty"`
 	ControlNamespace string               `json:"controlNamespace"`
+	Debug            bool                 `json:"debug,omitempty"`
```
Comment (pepov) [inline api/telemetry/v1alpha1/collector_types.go:25]: Can we add a description here on what it does and does not?

File: internal/controller/telemetry/otel_conf_gen.go [hunk 2]
```diff
@@ -93,18 +113,44 @@ func (cfgInput *OtelColConfigInput) generateOTLPExporters() map[string]any {
...
-		var otlpGrpcValues map[string]any
-		if err := yaml.Unmarshal(otlpGrpcValuesMarshaled, &otlpGrpcValues); err != nil {
-			return result
+		if output.Spec.OTLP != nil {
+
+			// TODO: add proper error handling
+			name := fmt.Sprintf("otlp/%s_%s", output.Namespace, output.Name)
+			otlpGrpcValuesMarshaled, err := yaml.Marshal(output.Spec.OTLP)
+			if err != nil {
+				return result
```
Comment (pepov) [inline internal/controller/telemetry/otel_conf_gen.go:122]: Can we at least log an error?

**Review by pepov — APPROVED** [review]
Looks great, just a few nits

File: go.mod [hunk 3]
```diff
@@ -1,98 +1,195 @@
...
 	github.com/cisco-open/operator-tools v0.34.0
-	github.com/onsi/ginkgo/v2 v2.13.0
-	github.com/onsi/gomega v1.29.0
-	github.com/open-telemetry/opentelemetry-operator v0.90.0
-	k8s.io/apimachinery v0.28.4
-	k8s.io/client-go v0.28.4
-	sigs.k8s.io/controller-runtime v0.16.3
+	github.com/onsi/ginkgo/v2 v2.16.0
+	github.com/onsi/gomega v1.31.1
+	github.com/open-telemetry/opentelemetry-operator v0.96.0
```
Comment (OverOrion) [inline go.mod:12]: Because of this I think `ENVTEST_OTEL_OPERATOR_VERSION` in the Makefile should be bumped as well.

**Review by OverOrion — APPROVED** [review]
LGTM, only version alignments needed

File: internal/controller/telemetry/otel_conf_gen.go (hunk 2 above)
Comment (pepov) [reply internal/controller/telemetry/otel_conf_gen.go]: we've discussed this offline and agreed to log an error here instead

**Pull request merged**